target/linux/rockchip/patches-5.4/111-rockchip-add-NanoPi-R4S-1Gb-DDR3.patch

```diff
@@ -0,0 +1,27 @@
+From dbbcf8b2cb8e321310510179caa12afb30525c17 Mon Sep 17 00:00:00 2001
+From: Marty Jones <mj8263788@gmail.com>
+Date: Sat, 12 Dec 2020 03:54:03 -0500
+Subject: [PATCH] rockchip: add NanoPi R4S 1Gb DDR3
+
```

Comment:
I don't get it.

If only the bootloader needs to be model-specific in order to initialize the DDR controller, what benefit does a specific kernel image provide at all?